waterdata/views.py

```diff
@@ -5,9 +5,40 @@
 from flask import render_template
 
 from . import app, __version__
+from .utils import get_water_services_data, parse_rdb
+
+
+water_services = app.config['WATER_SERVICES']
 
 
 @app.route('/')
 def home():
     """Render the home page."""
     return render_template('index.html', version=__version__)
+
+
+@app.route('/monitoring-location/<site_no>')
+def monitoring_location(site_no):
+    """
+    Monitoring Location view
+    :param site_no: USGS site number
+
+    """
+    rdb_resp_data = get_water_services_data(water_services, 'nwis/site/?site={}'.format(site_no))
```

Comment:
Consider setting format=rdb as a query parameter. We are expecting rdb, maybe we should make it explicit.